files/build_deploy.py

```diff
@@ -206,9 +213,9 @@ def notify_error(stage, error):
         notify_error('install', C.output)
 
     try:
-        syslog.syslog("Build of {}: bundle exec middleman build".format(name))
-        result = subprocess.check_output(['bundle', 'exec', 'middleman',
-                                          'build', '--verbose'])
+        command = builder_commands[config['builder']]
```

Comment:
Indeed, good catch. I am also wondering if we should assume that each step will always be 1 command, or it can be more.

And I am also pondering on putting that in a yaml file, like this:
$ cat /something/middleman.yml
- build:
  - "bundle exec middleman build --verbose"
- install
  - "bundle install"

Can we reasonably assume that bundle would be used to install if there is a Gemfile, or that's something that could be bound to change right now (like, is the state of the ecosystem such that everybody use it, if I am not wrong ?)